### ISSUE TITLE
Dashboard: Editor Settings; A11y: Google Tracking missing aria label

### ISSUE THREAD

**Issue by carloskelly13** [GoogleForCreators/web-stories-wp#4620]
## Bug Description

The Google Analytics Tracking field is missing a label. This was a regression, so a test to verify it exists would be great to add too.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
